openapi-bal-service/src/main/java/io/ballerina/openapi/converter/service/CacheConfigAnnotation.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package io.ballerina.openapi.converter.service;
+
+import java.util.ArrayList;
+import java.util.List;
+/**
+ * This class uses to store all the cache configuration details.
+ *
+ * @since 2.0.0
+ */
+public class CacheConfigAnnotation {
+    private boolean mustRevalidate = false;
+    private boolean noCache = false;
+    private boolean noStore = false;
+    private boolean noTransform = false;
+    private boolean isPrivate = false;
+    private boolean proxyRevalidate = false;
+    private int maxAge = -1;
```

Comment:
Also we set `max-age=3600` as default

Reply:
The same concern there as well, if the user doesn't provide any max-age value we need to add a default value to the generated string?
If in case the user set the negative value there could you please let me know whether we ignore it by adding the max-age field to the string or are we add `max-age` with the default value?

Reply:
**Case 1 :** If the `max-age` is not set in the annotation, then we add the **default** value `max-age=3600`.

**Case 2 :** If the `max-age` is set to a negative value, then we **do not** add `max-age` as a directive (ignore it in the string). This is how the user can disable the `max-age` directive.

**Case 3 :** If the `max-age` is set to a positive value, then we add that value as `max-age`.

Reply:
Fix with https://github.com/ballerina-platform/ballerina-openapi/pull/564/commits/880d1122b2fc897bb656d1f44c31bdf3a7906695